### PR TITLE
Add Disbursement and Sales Compensation queries

### PR DESCRIPTION
**Pull request by lderryck**
*Issue #, if available:*

*Description of changes:*
Adding Disbursement query and Sales Compensation query

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql [hunk 1]
```diff
@@ -72,7 +72,6 @@ agreements_with_uni_temporal_data as (
             *,
             ROW_NUMBER() OVER (PARTITION BY agreement_id, valid_from ORDER BY from_iso8601_timestamp(update_date) desc) as row_num
         from
-            -- TODO change to agreementfeed_v1 when Agreement Feed is GA'ed
```
Comment (vincent-larchet-aws) [inline seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql]: this comment should stay

File: seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql [hunk 2]
```diff
@@ -531,15 +530,13 @@ from invoiced_transactions inv
...
                                                                     and (inv.invoice_date_as_date >= acc_enduser.valid_from  and inv.invoice_date_as_date < acc_enduser.valid_to)
 
-    -- TODO left join because agreement feed is not GA yet and not 100% backfilled yet -> will be moved to INNER join after backfill
     left join agreements_with_history agg on agg.agreement_id = inv.agreement_id and (inv.invoice_date_as_date >= agg.valid_from  and inv.invoice_date_as_date < agg.valid_to)
-
     left join accounts_with_history_with_company_name acc_reseller on agg.proposer_account_id = acc_reseller.account_id
                                                                         and (inv.invoice_date_as_date >= acc_reseller.valid_from  and inv.invoice_date_as_date < acc_reseller.valid_to)
     -- if you want to get current offer name, replace the next join with: left join offer_targets_with_latest_revision_with_target_type off on agg.offer_id = off.offer_id
-    -- TODO left join because reseller's agreements show offer IDs not exposed in manufacturer's Offer Feed (yet, Nimish seeking Legal Approval)
```
Comment (vincent-larchet-aws) [inline seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql:540]: this comment should stay (but the very end in parenthesis)

**Review by vincent-larchet-aws — APPROVED** [review]
couple of nits founds (comments on comments ;-) ) -> approved and fix the comments in the next commit

File: seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql (hunk 1 above)
Comment (lderryck) [reply seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql]: Ok I've added this comment back.

File: seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql (hunk 2 above)
Comment (lderryck) [reply seller-data-feeds/queries/rev_recognition_invoice_bi_temporal_data.sql:540]: I've added the comment back minus the part in parentheses

**Review by vincent-larchet-aws — APPROVED** [review]

**Pull request merged**